network/nebula/nebula.yaml

```diff
@@ -0,0 +1,47 @@
+name: nebula
+depends:
+  - service: cri
+  - network:
+    - addresses
+    - connectivity
+    - etcfiles
+  - configuration: true
+container:
+  entrypoint: /usr/local/bin/nebula
+  args: 
+    - -config 
+    - /usr/local/etc/nebula/config.yml 
+  environment:
+  - PATH=/sbin:/usr/local/bin
+  security:
+    writeableRootfs: false
+    writeableSysfs: true
+  mounts:
+  # libs
+  - source: /lib
```

Comment:
if nebula is compiled as a static binary I guess these `lib` and `sbin` mounts are not needed

Reply:
Okie dokie, I'll remove the mounts